cot/src/form/fields/chrono.rs

```diff
@@ -118,11 +128,636 @@ impl SelectChoice for Weekday {
+    /// editable.
+    pub readonly: Option<bool>,
+
+    /// The increment (in seconds) between valid datetimes.
+    ///
+    /// Corresponds to the `step` attribute on `<input type="datetime-local">`.
+    /// If `None`, the browser’s default (60 seconds) is used. To override,
+    /// supply `Step::Value(Duration)` where `Duration` indicates the number
+    /// of seconds per “tick.”
+    ///
+    /// [`step` attribute]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local#step
+    pub step: Option<Step<Duration>>,
+
+    /// The timezone to use when parsing the browser‐supplied string.
+    ///
+    /// Browsers send a naive "YYYY-MM-DDThh:mm" (no offset). If `timezone` is
+    /// `Some(tz)`, we convert that naive time into a `DateTime<FixedOffset>`
+    /// using `tz.from_local_datetime(...)`. The timezone should be a
+    /// [`Tz`] which is capable of handling true DST transitions and
+    /// timezone rules. If `None`, defaults to UTC.
+    pub timezone: Option<Tz>,
+
+    /// Choose how to handle ambiguous local time conversion (e.g. during a DST
+    /// fall-back).
+    ///
+    /// - `Some(true)`:  always pick the **later** of the two possible instants
+    ///   (DST time).
+    /// - `Some(false)`: always pick the **earlier** of the two possible
+    ///   instants (standard time).
+    /// - `None`:         treat an ambiguous local time as a validation error.
+    pub prefer_latest: Option<bool>,
+}
+
+impl Display for DateTimeWithTimezoneField {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        let mut hidden_tag = HtmlTag::input("hidden");
+
+        // https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/input/datetime-local#setting_timezones
+        hidden_tag
+            .attr("name", format!("__{}_timezone", self.id()))
+            .attr("id", format!("__{}_timezone", self.id()));
+
+        let mut dt_tag = HtmlTag::input("datetime-local");
+        dt_tag.attr("name", self.id());
+        dt_tag.attr("id", self.id());
+
+        if self.options.required {
+            dt_tag.bool_attr("required");
+        }
+        if let Some(max) = self.custom_options.max {
+            dt_tag.attr(
+                "max",
+                max.naive_local()
+                    .format(BROWSER_DATETIME_WITHOUT_SEC_FMT)
+                    .to_string(),
+            );
+        }
+        if let Some(min) = self.custom_options.min {
+            dt_tag.attr(
+                "min",
+                min.naive_local()
+                    .format(BROWSER_DATETIME_WITHOUT_SEC_FMT)
+                    .to_string(),
+            );
+        }
+
+        if let Some(readonly) = self.custom_options.readonly {
+            if readonly {
+                dt_tag.bool_attr("readonly");
+            }
+        }
+
+        if let Some(value) = &self.value {
+            dt_tag.attr("value", value);
+            let tz = DateTime::parse_from_str(value, "%Y-%m-%dT%H:%M %z")
```

Comment:
Where is this format coming from? Would it make more sense to use ISO 8601 instead? (`"%Y-%m-%dT%H:%M%:z"`)

(not sure if I understand this, perhaps it has some connection with the hidden field I've mentioned)